src/sz.c

```diff
@@ -106,6 +107,7 @@ sz_boot_size2index_tab(const sc_data_t *sc_data) {
 
 void
 sz_boot(const sc_data_t *sc_data) {
+	sz_large_pad = opt_cache_oblivious ? PAGE : 0;
```

Comment:
Should take cache_oblivious as an argument rather than looking at the global -- this is new-style code where where we're trying to minimize dependencies on global state.